packages/shared/src/features/onboarding/steps/FunnelOrganicSignup.tsx

```diff
@@ -171,75 +180,132 @@ export const FunnelOrganicSignup = withIsActiveGuard(
+                onSuccessfulLogin={() => {
+                  // user now is logged, even if the `user` object is not populated yet.
+                  // this callback is fired only after a lot of auth checks
+                  onTransition?.({
+                    type: FunnelStepTransitionType.Complete,
+                    details: { user },
+                  });
                 }}
-                title={headline}
-                description={explainer}
               />
-            )}
-            <AuthOptions
-              {...staticAuthProps}
-              defaultDisplay={
-                isSocialSignupActive
-                  ? AuthDisplay.SocialRegistration
-                  : authDisplay
-              }
-              formRef={formRef}
-              onboardingSignupButton={{
-                size: isMobile ? ButtonSize.Medium : ButtonSize.Large,
-                variant: ButtonVariant.Primary,
-              }}
-              onSuccessfulRegistration={onSuccessfulRegistration}
-              onAuthStateUpdate={onAuthStateUpdate}
-              onSuccessfulLogin={() => {
-                // user now is logged, even if the `user` object is not populated yet.
-                // this callback is fired only after a lot of auth checks
-                onTransition?.({
-                  type: FunnelStepTransitionType.Complete,
-                  details: { user },
-                });
-              }}
-            />
+            </div>
+            <div className="flex flex-1 tablet:ml-auto tablet:flex-1 laptop:max-w-[37.5rem]" />
           </div>
-          <div className="flex flex-1 tablet:ml-auto tablet:flex-1 laptop:max-w-[37.5rem]" />
-        </div>
+        </ConditionalWrapper>
+
         {!!src && (
-          <img
-            alt="Onboarding background"
-            aria-hidden
-            className={classNames(
-              'pointer-events-none absolute inset-0 -z-1 size-full object-cover transition-opacity duration-150',
-              { 'opacity-[.24]': isEmailSignupActive },
-            )}
-            fetchPriority="high"
-            loading="eager"
-            role="presentation"
-            src={src}
-            srcSet={`${src} 450w, ${srcDesktop || src} 1024w`}
-            sizes="(max-width: 655px) 450px, 1024px"
-          />
+          <picture>
+            <source media="(max-width: 655px)" srcSet={src} />
+            <source media="(min-width: 656px)" srcSet={srcDesktop || src} />
+            <img
+              alt="Onboarding background"
+              aria-hidden
+              className={classNames(
+                'pointer-events-none absolute inset-0 -z-1 size-full object-cover transition-opacity duration-150',
+                { 'opacity-[.24]': isEmailSignupActive },
+              )}
+              fetchPriority="high"
+              loading="eager"
+              role="presentation"
+              src={src}
+            />
+          </picture>
```

Comment:
Moved from srcSet + sizes to picture since it was bugged. Now works like a charm!